cmd/template/cluster/provider/templates/capa/types.go

```diff
@@ -23,8 +24,17 @@ type AWS struct {
 }
 
 type Network struct {
-	AvailabilityZoneUsageLimit int    `json:"availabilityZoneUsageLimit,omitempty"`
-	VPCCIDR                    string `json:"vpcCIDR,omitempty"`
+	AvailabilityZoneUsageLimit int      `json:"availabilityZoneUsageLimit,omitempty"`
+	VPCCIDR                    string   `json:"vpcCIDR,omitempty"`
+	TopologyMode               string   `json:"topologyMode,omitempty"`
+	VPCMode                    string   `json:"vpcMode,omitempty"`
+	ApiMode                    string   `json:"apiMode,omitempty"`
```

Comment:
Either all `ABBREVIATIONCamelCase` or `AbbreviationCamelCase`. In Go, it's typically `APIMode`, not `ApiMode`, since the abbreviation is all-capital "API" (same as words XML, VPC, DNS, etc.).